meetings/2022-12-22.md

```diff
@@ -53,6 +57,51 @@ URL: https://github.com/solid/notifications/pull/135
 
 >ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.
 
+* RG: We didn't specify how does the request and response work when you want to modify the subscription
+* SC: Not in this PR, we still didn't reach consensus with this feature. We might need an issue for it.
+* RG: I think we have an issue for it.
+* eP: is there something problematic in 135 if we don't include updating now?
+* RG: No but it will have to do with the data model (eventually). I don't see a problem as another thing but it should be part of the next release.
+* SC: That feature would require both an update to the protocol and data model.
+* eP: RG, do you have a specific proposal? We can take care of this PR and then create issue to work together towards a PR for updating the subscription. Still need a clear approach.
+
+ACTION: RG to check existing issues or create an issue to track updating subscriptions.
+RG: https://github.com/solid/notifications/issues/103
+
+* SC: I tried to incorporate all of the comments. I think there are 2 or 3 which I still didn't get to.
+* ...: Pavlik suggested we could get rid of the terminology.
+* eP: In general, terminology makes sense if there is no dedicated section and a term used throughout the spec. On the contrary all the terms we have in terminology they have their own section. In that case those sections re normative and we can reuse. It could clean-up linking.
+* SC: I suggest we walk through each term and see. I'm +0.75 to removing if they can belong elsewhere.
+* SC: Let's dive into it using my screen share
+* SC: There is also the issue of clarifying when the subscription step is needed
+* eP: I'm ok to have it as dedicated PR
+* SC: We can still try to get it in 135
+* SC: I also pointed out possibly collision of startTime with AS JSON-LD context mapping
+
+ACTION: Issue for https://github.com/solid/notifications/pull/135#issuecomment-1361127017
+
+* eP: https://www.w3.org/TR/2012/REC-owl2-primer-20121211/#Property_Characteristics
+> Note that this statement does not require every individual to have a husband, it just states that there can be no more than one. Moreover, if we additionally had a statement that Mary's husband is James and another that Mary's husband is Jim, it could be inferred that Jim and James must refer to the same individual.
+
+* eP: If channel can only have one startTime. If there are more than one, it would be invalid. In practice, functional property definition, two start times entails they are the same. Not sure if we can use FunctionalObjectProperty and DataTypeProperty together ... It is safe for now and have separate issue. I think it is wrong definition in the AS2 vocab.
+* SC: Maybe we consider using start/endDateTime in notifications JSON-LD context.
```

Comment:
```suggestion
* eP: Channel can only have one `startTime`. If there is more than one, it would be invalid. In practice, functional property definition: two `startTime`s entails they are the same. Not sure if we can use `FunctionalObjectProperty` and `DataTypeProperty` together ... It is safe for now and have separate issue. I think it is wrong definition in the AS2 vocab.
* SC: Maybe we consider using `startDateTime`/`endDateTime` in notifications JSON-LD context.
```